kore/src/Kore/Repl/Data.hs

```diff
@@ -688,31 +621,21 @@ makeKoreReplOutput :: String -> ReplOutput
 makeKoreReplOutput str =
     ReplOutput . return . KoreOut $ str <> "\n"
 
-runUnifierWithExplanation ::
+runUnifierWithoutExplanation ::
     forall m a.
     MonadSimplify m =>
-    UnifierWithExplanation m a ->
-    m (Either ReplOutput (NonEmpty a))
-runUnifierWithExplanation (UnifierWithExplanation unifier) =
-    failWithExplanation <$> unificationResults
+    UnifierT m a ->
+    m (Maybe (NonEmpty a))
```

Comment:
Why was this changed to a `Maybe`?

Reply:
I thought there is no need for `ReplOutput` because there would be no `explainBottom`. The explanations would be printed as log entries instead of being wrapped in a `Left`

Reply:
What happens if the result is `Nothing`? Do we just not print anything anymore?

Reply:
Yes, [here](https://github.com/kframework/kore/pull/2674/files#diff-b87f2e448b6098a71a0953983f8941bae7a8d3f93aa3284c54ce2cb1d7a3efc5R1619). I don't know any means to tell if the log entry was called or not in order to tell the user "No explanation given" in the appropriate case.

Reply:
Ok, I understand now. Thanks for explaining.